cifs-utils.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: cifs-utils
+  version: 7.0
+  epoch: 0
+  description: "CIFS filesystem user-space tools"
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - libcap-dev
+      - keyutils-dev
+      - krb5-dev
+      - autoconf
+      - automake
+      # - samba-dev
```

Comment:
Why is this commented out?

Reply:
I looked at the packages specified in the [APKBUILD](https://git.alpinelinux.org/aports/tree/main/cifs-utils/APKBUILD#n10) the `samba-dev` is one of them but it seems quite complicated so I tried to build this package without it to see whether we depend on `samba-dev`. At a result, it seems we don't. I also checked [here](https://gitlab.archlinux.org/archlinux/packaging/packages/cifs-utils/-/blob/main/PKGBUILD?ref_type=heads), there is no `samba-dev`. PTAL @kaniini

Reply:
We definitely need `samba` for `cifs-utils` to work in a meaningful way.